sandy/njoy.py

```diff
@@ -951,12 +1004,17 @@ def process(
         kwargs["temp"] = temperatures[0]
         text += _groupr_input(-e, -p, -v, **kwargs)
         o = 32
-        text += _moder_input(-v, o)
+        if errorr is False:
+            text += _moder_input(-v, o)
         outputs[f"tape{o}"] = join(
             wdir,
             f"{outprefix}{tag}.gendf",
             )
     if errorr:
+        if groupr:
+            kwargs["ngout"] = -v
```

Comment:
I am not sure ngout should be given as a kwarg...it doesn't seem consistent with the other tape inouts/outputs.
You could do something like
```
p_ = 0 if groupr else p
v_ = v if groupr else 0
errorr_input(-e, p_, -v_, o, **kwargs)
```

Reply:
Ia not sure `v` is a good variable name.
- `p` is for pending
- `e` is for endf
- `o` is for output
Then I would call it `g` for gendf